tests/calc/test_indices.py

```diff
@@ -185,6 +185,10 @@ def test_corfidi_motion():
                   38.32734963, -11.90040377] * units('kt')
     assert_almost_equal(motion_full.flatten(), truth_full, 8)
 
+
+def test_corfidi_motion_override_llj():
+    """Test corfidi MCS motion with overridden LLJ"""
```

Comment:
Minor style nit:
```suggestion
    """Test corfidi MCS motion with overridden LLJ."""
```

Reply:
pylint caught that and I fixed it in 50e6b6b

Reply:
👍